src/routes/admin/existShopDataPUT.js

```diff
@@ -0,0 +1,110 @@
+const responseMessage = require('../../constants/responseMessage');
+const statusCode = require('../../constants/statusCode');
+const { adminUserDB, shopDB, themeDB, categoryDB, userDB } = require('../../db');
+const db = require('../../db/db');
+const util = require('../../lib/util');
+const jwtHandlers = require('../../lib/jwtHandlers');
+const slackAPI = require('../../middlewares/slackAPI');
+const shopDataGET = require('./shopDataGET');
+const { duplicatedDataClean } = require('../../lib/convertRawDataToProccessedData');
+
+module.exports = async (req, res) => {
+  const { shopName, subway, roadAddress, landAddress, time, close, phone, homepage, instagram, blog, store, area, category, theme} = req.body;
+  if(!shopName || !subway || !roadAddress || !landAddress || !area || !category || !theme){
+      return res.status(statusCode.BAD_REQUEST).send(util.fail(statusCode.BAD_REQUEST, responseMessage.NULL_VALUE));
+  }
+  
+  const imageUrls = req.imageUrls;
+  if(imageUrls.length === 0) {
+      return res.status(statusCode.BAD_REQUEST).send(util.fail(statusCode.BAD_REQUEST,responseMessage.NO_IMAGE));
+  }
+  
+  let shopId;
+  let client;
+    try {
+        client = await db.connect(req);
+
+        //shopName 통해서 검색해서 shopId 받아오기 
+        if(shopName){
+            let shopArr = await shopDB.getShopByName(client, shopName);
+            if(shopArr){
+                shopArr = duplicatedDataClean(shopArr,'shopId','category');
+                shopId = await shopArr[0].shopId;
+            }
+        }
+
+        // 1.테마 관련
+        //새로운 테마를 받아와서 테마번호 배열로 바꿔주기 
+        let allTheme = await themeDB.getAllTheme(client);
+        const newTheme = [];
+        allTheme.forEach((element) => {
+            if(theme.includes(element.name)){
```

Comment:
```suggestion
            if(JSON.parse(theme).includes(element.name)){
```
요런 식으로 해야 "["빈티지","아기자기"]" 으로 들어왔던 데이터가 ["빈티지","아기자기"]로 파싱될 거 같아요!